### ISSUE TITLE
Starting wg-ui fails on Raspberry Pi

### ISSUE THREAD

**Issue by Brainscrewer** [EmbarkStudios/wg-ui#91]
**Describe the bug**
First of all, I am aware that with the release of version 1.1. ARM builds are also supported. However, running the supplied example command in the README on my raspberry Pi still results in docker failing to start the container.

```
pi@raspberrypi:~/test $ sudo docker run --rm -it --privileged --entrypoint "/wireguard-ui" -v /tmp/wireguard-ui:/data -p 8080:8080 -p 5555:5555 embarkstudios/wireguard-ui:latest --data-dir=/data --log-level=debug
standard_init_linux.go:211: exec user process caused "no such file or directory"

```
**To Reproduce**
Run the above command

**Expected behavior**
Container deploying

**Device:**
- Raspberry Pi 3 Model B+

**Comment by ahsandar** [comment]
```docker run --rm -it -e WIREGUARD_UI_LOG_LEVEL=debug embarkstudios/wireguard-ui:latest
standard_init_linux.go:211: exec user process caused "no such file or directory"```

same behaviour on `Raspberry PI 4`

**Comment by monxas** [comment]
Same behavior here. still no solution?

**Comment by janispritzkau** [comment]
I think it's missing the ARMv8 build which is also the architecture my Odroid C2 uses.

**Comment by Schrolli91** [comment]
Yes its the missing ARMv8 - same problem here at a RPi 4B 4Gb

**Comment by tuxflo** [comment]
The solution is fairly simple... just build the image on you own. Its just `docker build -t wg_ui .`  from the cloned Repo. Then replace the image name of the `docker run` command with your tag `wg_ui`.

**Comment by ahsandar** [comment]
don't think its that straightforward otherwise someone would have done that and uploaded an image to the hub. There is no build support for armv8 in the Makefile even

https://github.com/EmbarkStudios/wg-ui/blob/master/Makefile#L38

**Comment by tuxflo** [comment]
Well I tried it and it works fine on my Pi 4.

**Comment by suom1** [comment]
> don't think its that straightforward otherwise someone would have done that and uploaded an image to the hub. There is no build support for armv8 in the Makefile even
> 
> https://github.com/EmbarkStudios/wg-ui/blob/master/Makefile#L38

If you run `make build` on a Pi 4, it will build it for Pi 4. 
We have just added cross-compile support if you would want to build for other platforms instead for the ones you're running at the moment.

We do build docker images for `linux/arm64`, which should work for ARMv8. However sadly I myself do not have access to a device like that to do testing and developing on, if you have one and would like to help the project we would be more than happy! 

I have added build instructions for ARMv8 in #109 if you desire to build ARMv8(arm64) binaries on something that is not ARMv8.
